test/test_distributions.py

```diff
@@ -502,20 +502,12 @@ def is_all_nan(tensor):
     ]),
     Example(TransformedDistribution, [
         {
-            'base_distribution': Normal(variable([1, 1], requires_grad=True),
-                                        variable([0, 1], requires_grad=True)),
-            'transforms': [],
-        },
-        {
-            'base_distribution': Normal(variable([1, 1], requires_grad=True),
-                                        variable([-1, -1], requires_grad=True)),
-            'transforms': ExpTransform(),
+            'base_distribution': Normal(0, 1),
+            'transforms': lambda x: x,
```

Comment:
Cannot this be `identity_transform`?

Reply:
This is actually intended to be a `BAD_EXAMPLE` where `identity_transform` should be used in stead of `lambda x: x`. This test checks that `TransformedDistribution.__init__()` complains that its argument is not a `Transform` :wink: